docker-compose.yml

```diff
@@ -4,7 +4,7 @@ services:
       context: .
     image: mongomock:latest
     working_dir: /project
-    command: hatch test
+    command: sleep infinity
```

Comment:
I suppose this was needed for manual testing?

Reply:
Done.